### PR TITLE
Used const variable in scheduler test.

### PR DESCRIPTION
**Pull request by gyliu513**
**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
none
```

### CODE REVIEW

File: plugin/pkg/scheduler/core/generic_scheduler_test.go [hunk 1]
```diff
@@ -410,7 +410,7 @@ func TestHumanReadableFitError(t *testing.T) {
 			"3": []algorithm.PredicateFailureReason{algorithmpredicates.ErrNodeUnderDiskPressure},
 		},
 	}
-	if strings.Contains(error.Error(), "No nodes are available that match all of the following predicates") {
+	if strings.Contains(error.Error(), NoNodeAvailableMsg) {
```
Comment (k82cn) [inline plugin/pkg/scheduler/core/generic_scheduler_test.go]: Can you also help to remove `:` in `NoNodeAvailableMsg`? It's unnecessary to me :).

Comment (gyliu513) [inline plugin/pkg/scheduler/core/generic_scheduler_test.go]: Done

**Comment by k82cn** [comment]
/lgtm
/approve no-issue

Just const var replacement, so I'll get that merged :).

**Pull request merged**